src/controllers/userController.ts

```diff
@@ -0,0 +1,81 @@
+import { prisma } from '@/model/prisma';
+import { Body, errorCode } from '@/utils/utils';
+import { DBError } from '@/validation/custom-error';
+
+export const user = {
+  getById: async (id: number) => {
+    if (isNaN(id)) {
+      throw new DBError('UserId should be a number.', errorCode.INVALID);
+    }
+    try {
+      const user = await prisma.user.findUnique({
+        where: { id: id },
+      });
+
+      if (!user) {
```

Comment:
Do you really need this catch? So if a user is not found you catch an error and then throw it again, right?